src/NServiceBus.AzureStoragePersistence.AcceptanceTests/app.config

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
```

Comment:
We're going to be killing the AcceptanceTests Project as per an idea that Andreas had for dealing with testing on Persisters. It won't happen until we get this first pass at the upgrade done though.